Framework/TestHelpers/src/InstrumentCreationHelper.cpp

```diff
@@ -83,6 +83,46 @@ void addFullInstrumentToWorkspace(MatrixWorkspace &workspace, bool includeMonito
   workspace.setInstrument(instrument);
 }
 
+void addInstrumentWithGeographicalDetectorsToWorkspace(Mantid::API::MatrixWorkspace &workspace, const int nlat,
+                                                       const int nlong, const double anginc,
+                                                       const std::string &instrumentName) {
+  V3D samplePosition(0., 0., 0.);
+  V3D sourcePosition(0., 0., -14.);
+
+  Instrument_sptr instrument = std::make_shared<Instrument>(instrumentName);
+  instrument->setReferenceFrame(
+      std::make_shared<ReferenceFrame>(Mantid::Geometry::Y, Mantid::Geometry::Z, Right, "0,0,0"));
+
+  InstrumentCreationHelper::addSource(instrument, sourcePosition, "source");
+  InstrumentCreationHelper::addSample(instrument, samplePosition, "sample");
+
+  // set up detectors with even spacing in latitude and longitude (to match geographical angles
+  // approach used in the spatial interpolation\sparse instrument functionality)
+  int i = 0;
+  constexpr double deg2rad = M_PI / 180.0;
+  const double angincRad = anginc * deg2rad;
+  auto R = 1.0;
```

Comment:
Probably not overly relevant since it's just a move, but it seems like `R` could be `constexpr`?